src/main/java/com/github/manolo8/darkbot/backpage/GalaxyManager.java

```diff
@@ -0,0 +1,92 @@
+package com.github.manolo8.darkbot.backpage;
+
+import com.github.manolo8.darkbot.Main;
+import com.github.manolo8.darkbot.backpage.entities.galaxy.*;
+import com.github.manolo8.darkbot.utils.XmlHelper;
+import org.dom4j.Document;
+import org.dom4j.Element;
+import org.dom4j.io.SAXReader;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class GalaxyManager {
+
+    public GalaxyInfo galaxyInfo;
+    private BackpageManager backpageManager;
+    private Main main;
+
+    public GalaxyManager(Main main, BackpageManager backpageManager) {
+        this.main = main;
+        this.backpageManager = backpageManager;
+        this.galaxyInfo = new GalaxyInfo();
+    }
+
+    public void performGateSpin(String gateName, boolean sample, int gateId, int multiplier, int minWait) {
```

Comment:
You should probably document this
what is sample? 
why do i need a gateId if i'm passing a gateName?
shouldn't multiplier be a boolean and just use whatever multiplier was returned in last response from the server?